app/Models/Stakeholders/Department.php

```
@@ -33,6 +33,10 @@
 
     public static function getByCode($name)
     {
-        return self::where('name', trim($name))->first()->code;
+        if (!$name) {
+            return 'ACS';
+        }else {
+            return self::where('name', trim($name))->first()->code;
+        }
```

PHPMD (2.15.0)

The method getByCode uses an else expression. Else clauses are basically not necessary and you can simplify the code by not using them. (Clean Code Rules)

(ElseExpression)